### DISCUSSION TITLE
Preconfiguring Installation

### DISCUSSION THREAD

**Discussion by DE-mbecker** [requarks/wiki#4658] [Help / Questions]
Hi,

we are provisioning wikijs to our docker host via Ansible. We would like to preconfigure the environment in ansible, for example set the base url, set admin user and password.

We can do this with sql scripts and importing them into the postgresql database. Is there no way to do this via cli or environment variables?

Greetings
Marc

**Comment by openbrian** [comment]
@DE-mbecker Have you had any luck with installing/configuring Wiki.js with Ansible?

I've heard in other conversations that base url is not happening.  I tried myself.  It's a hard problem to solve.

**Reply by DE-mbecker** [reply]
I did not find a solution for automatic configuration and I do not spent much time on this, because we don‘t use wikijs anymore.
